diff --git a/node_modules/deepfilternet3-noise-filter/dist/index.js b/node_modules/deepfilternet3-noise-filter/dist/index.js
index a841dbd..28953c3 100644
--- a/node_modules/deepfilternet3-noise-filter/dist/index.js
+++ b/node_modules/deepfilternet3-noise-filter/dist/index.js
@@ -67,7 +67,7 @@ class WorkerManager {
             WorkerManager.workerReadyResolve = resolve;
         });
         const worker = new Worker(new URL('./DeepFilterWorker.js', (typeof document === 'undefined' ? require('u' + 'rl').pathToFileURL(__filename).href : (_documentCurrentScript && _documentCurrentScript.tagName.toUpperCase() === 'SCRIPT' && _documentCurrentScript.src || new URL('index.js', document.baseURI).href))), {
-            type: this.config.type,
+            type: 'classic',
             name: this.config.name
         });
         worker.onmessage = (event) => this.handleWorkerMessage(event);
